package.json

```diff
@@ -58,7 +58,7 @@
       }
     },
     "start": {
-      "command": "babel-node bin/server",
+      "command": "npm run lint && babel-node bin/server",
```

Comment:
Let me know if there is a better way to do this. I was surprised to see the linter is not set to run on start, especially as there were already 4 errors being thrown.